moped-database/views/uniform_features.sql

```diff
@@ -132,9 +105,7 @@ LEFT JOIN (
     SELECT
         d.feature_id,
         array_agg(d.council_district_id) AS council_districts
-    FROM features_council_districts AS d
+    FROM features_council_districts d
     GROUP BY d.feature_id
-) AS districts ON feature_drawn_lines.id = districts.feature_id
+) districts ON districts.feature_id = feature_drawn_lines.id
 WHERE feature_drawn_lines.is_deleted = FALSE;
-
-COMMENT ON VIEW uniform_features IS 'This view unifies various geographical feature data from multiple tables such as signals, street segments, intersections, drawn points, and lines. It provides a view of these features along with their attributes, geographic details, and council district associations.';
```

Comment:
Would it make sense to think about extending this to capture these comments too? Not 100% but it feels like the intention here was to make sure the comment survived future drops followed by creating again.

Reply:
Yea, it totally should make sense for us to think about it, and thank you for tagging this. I was thinking about it on the way back from the apple store today, and I think the best way to do this is drop the `psql` SQL based extraction of the query driving views, and transition to `pg_dump` which is the more capable, albeit, harder to use tool if you're seeking succinct results.

Doing this is hard enough that I think maybe I'd suggest we spin it off as a future enhancement. 

But - it bears saying - you're 100% right that this will cause us to loose comments on the tables without careful intervention by the dev doing the view update. Not ideal.

Reply:
To make sure I'm understanding, in order to not lose the comments, devs will have to go and find the comment from the most recent up migration, and add it to the new migration? 

Only some of our views have comments, its going to be hard to remember which ones we need to go back and find comments for.

Reply:
Yea - you're right on Chia with your understanding. The way it's configured now, it's just a matter of time until we loose the comments on the views. There's a way to do it right with `pg_dump`, and I'm eager to get that worked out. We can spin that off into an issue to pick up the benefits of this work now, or I would be happy to roll that work into this PR -- either way is great with me.